packages/backend/package.json

```diff
@@ -43,6 +45,9 @@
     "typescript": "^3.9.2"
   },
   "nodemonConfig": {
-    "watch": "./dist"
+    "watch": [
+      "./dist",
+      "node_modules/@backstage*"
```

Comment:
If this works as intended it's gonna watch a lot of files, maybe something that ends with `/src` instead? :D

Reply:
Hah, i see that that actually has no matches at all :) I'll just revert this change